babel.config.cjs

```diff
@@ -5,34 +5,47 @@
 const ignore = require('./webpack/babel-ignored.cjs');
 
 const BROWSER_TARGETS = {
-  chrome: '49',
-  edge: '14',
-  firefox: '52',
-  safari: '9.0',
-};
-
-const MODERN_BROWSER_TARGETS = {
-  chrome: '106',
-  firefox: '105',
-  safari: '15.0',
+  production: {
+    chrome: '125',
+    edge: '125',
```

Comment:
>  * Chrome and Edge versions 125 (released in May 2024).

According to [Chrome release calendar](https://chromiumdash.appspot.com/schedule), Chrome 125 doesn’t have _Extended Support_ (which is apparently followed by the _Extended Stable Channel_ of [Microsoft Edge releases](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-release-schedule)), contrary to the version 128 of Firefox version (release on July 9, 2024) you pinned.

I guess that you just picked up a one-year old version (unless there is another reason I’m missing).

Please choose a one-year old Chrome/Edge version with Extended Support/Extended Stable Channel.

```suggestion
    chrome: '126',
    edge: '126',
```